packtools/sps/models/funding_group.py

```diff
@@ -24,15 +24,26 @@ class FundingGroup:
     def __init__(self, xmltree):
         self._xmltree = xmltree
 
+    @property
+    def financial_disclosure(self):
+        # TODO separar os valores entre 'funding-source' e 'award-id'
+        items = []
+        for node in self._xmltree.xpath(".//fn-group/fn[@fn-type='financial-disclosure']/p"):
+            link = node.xpath(".//ext-link[@ext-link-type='uri']")
+            if link:
+                items.append(link[0].get("{http://www.w3.org/1999/xlink}href"))
+            else:
+                items.append(node.text)
```

Comment:
@Rossi-Luciano o dado não deve perder a semântica. Ao colocar o link e o texto encontrado com itens em uma lista você perde o que é link e o que é texto

Reply:
@Rossi-Luciano pensando melhor... me parece que o problema é que você está prevendo somente ext-link, que não é uma abordagem genérica, e tende a ter perda de conteúdo retornado se a estrutura do fn for mais complexa. Então, para uma abordagem mais genérica, seria o caso de retornar todos os `text()` de dentro de `fn[@fn-type='financial-disclosure']`. Ou seja:
Tente isso
```python
for node in self._xmltree.xpath(".//fn-group/fn[@fn-type='financial-disclosure']"):
            yield " ".join(node.xpath("//text()"))
```
Isso deve dar o mesmo resultado que o seu teste, com a diferença que retorna um gerador no lugar de lista.
Para obter lista use list() no retorno de financial_disclosure.